e2e/pages/adf/contentServicesPage.ts

```diff
@@ -229,459 +221,396 @@ export class ContentServicesPage {
     }
 
-    getAttributeValueForElement(elementName, propertyName) {
+    async getAttributeValueForElement(elementName, propertyName): Promise<string> {
         const elementSize = element(by.css(`.adf-document-list-container div.adf-datatable-cell[data-automation-id="${elementName}"][title="${propertyName}"] span`));
-        return BrowserActions.getText(elementSize);
-    }
-
-    checkMenuIsShowedForElementIndex(elementIndex) {
-        const elementMenu = element(by.css(`button[data-automation-id="action_menu_${elementIndex}"]`));
-        BrowserVisibility.waitUntilElementIsVisible(elementMenu);
+        return await BrowserActions.getText(elementSize);
     }
 
-    navigateToCardFolder(folderName) {
-        BrowserActions.closeMenuAndDialogs();
-        const folderCard = element(by.css(`.adf-document-list-container div.adf-image-table-cell.adf-datatable-cell[data-automation-id="${folderName}"]`));
-        folderCard.click();
-        const folderSelected = element(by.css(`.adf-datatable-row.adf-is-selected div[data-automation-id="${folderName}"].adf-datatable-cell--image`));
-        BrowserVisibility.waitUntilElementIsVisible(folderSelected);
-        browser.actions().sendKeys(protractor.Key.ENTER).perform();
+    async checkMenuIsShowedForElementIndex(elementIndex): Promise<void> {
+        const elementMenu: ElementFinder = element(by.css(`button[data-automation-id="action_menu_${elementIndex}"]`));
+        await BrowserVisibility.waitUntilElementIsVisible(elementMenu);
     }
 
-    getGridViewSortingDropdown() {
-        const sortingDropdown = element(by.css('mat-select[data-automation-id="grid-view-sorting"]'));
-        BrowserVisibility.waitUntilElementIsVisible(sortingDropdown);
-        return sortingDropdown;
+    async navigateToCardFolder(folderName): Promise<void> {
+        await BrowserActions.closeMenuAndDialogs();
+        const folderCard: ElementFinder = element(by.css(`.adf-document-list-container div.adf-image-table-cell.adf-datatable-cell[data-automation-id="${folderName}"]`));
+        await BrowserActions.click(folderCard);
+        const folderSelected: ElementFinder = element(by.css(`.adf-datatable-row.adf-is-selected div[data-automation-id="${folderName}"].adf-datatable-cell--image`));
+        await BrowserVisibility.waitUntilElementIsVisible(folderSelected);
+        await browser.actions().sendKeys(protractor.Key.ENTER).perform();
     }
 
-    selectGridSortingFromDropdown(sortingChosen) {
-        BrowserActions.closeMenuAndDialogs();
-        const dropdownSorting = this.getGridViewSortingDropdown();
-        BrowserActions.click(dropdownSorting);
-        const optionToClick = element(by.css(`mat-option[data-automation-id="grid-view-sorting-${sortingChosen}"]`));
-        BrowserActions.click(optionToClick);
+    async selectGridSortingFromDropdown(sortingChosen): Promise<void> {
+        // await BrowserActions.closeMenuAndDialogs();
```

Comment:
remove comment if not needed

Reply:
from the blame has been committed by you , I usually not comment code ;)

Reply:
true, none the less, should be removed :D